Fix typos

thepicture opened the pull request:
### Description

- addEventLister -> addEventListener (window/error_event)
- my_obj -> myObj (object/getownpropertynames)

wbamberg reviewed (approved):
👍 thank you @thepicture !

thepicture committed:
Fix typos

addEventLister -> addEventListener (window/error_event)
my_obj -> myObj (object/getownpropertynames)
